mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -4023,7 +4033,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n  \"data\": {\n    \"type\": \"providers\",\n    \"attributes\": {\n      \"providerToken\": {\n        \"value\": 123\n      },\n      \"proxy\": {\n        \"id\": \"<n>\"\n      }\n    }\n  }\n}"
+											"raw": "{\n  \"data\": {\n    \"type\": \"providers\",\n    \"attributes\": {\n      \"providerToken\": {\n        \"value\": 123\n      },\n      \"packagesSelected\": 1,\n      \"proxy\": {\n        \"id\": \"<n>\"\n      }\n    }\n  }\n}"
```

Comment:
Is "packagesSelected" a new attribute that we are getting back new from RM API?

Reply:
with respect to changes done in https://issues.folio.org/browse/MODKBEKBJ-182 we send an update request to RM API only if `packagesSelected` exists and have a value, so, to check the error response I have updated the body of those requests

Reply:
ok, sounds good